ingestion/src/metadata/orm_profiler/api/workflow.py

```diff
@@ -67,6 +72,25 @@ def __init__(self, config: OpenMetadataWorkflowConfig):
             self.config.workflowConfig.openMetadataServerConfig
         )
 
+        if not self.is_sample_source(self.config.source.type):
+            # We override the current serviceConnection source object if source workflow service already exists in OM.
+            # We retrieve the service connection from the secrets' manager when it is configured. Otherwise, we get it
+            # from the service object itself.
+            service_type: ServiceType = get_service_type_from_source_type(
+                self.config.source.type
+            )
+            metadata = OpenMetadata(config=self.metadata_config)
```

Comment:
we are defining this already line 114. should we just remove it and instead call `self.metadata...`